src/selector.js

```diff
@@ -0,0 +1,20 @@
+export const PATH_KEY = Symbol.for('i18next/PATH_KEY');
```

Comment:
Is there any benefit for having this Symbol to be global and exported?
instead of:
`const PATH_KEY = Symbol('i18next/PATH_KEY')`

Reply:
Good call -- fixed in 156646d988957486371e754a2255da18110f31ea

Reply:
Fixed in 2b61265fb757db7548a8b2c2705ae9a8b62290b3